apps/trading/client-pages/markets/closed.tsx

```diff
@@ -250,17 +251,25 @@ const ClosedMarketsDataGrid = ({
     ];
   }, [chainId, t]);
 
+  const gridStore = useMarketsStore((store) => store.gridStore);
```

Comment:
I dont see loads of value of persisting grid state for these grids and would prefer to leave it out unless someone has specifically argued a case to have it

Reply:
cool. removed them for now.